slack_bot/app.py

```diff
@@ -67,11 +67,33 @@ def is_today_holiday() -> bool:
 
 
 def get_token() -> str:
-    return os.environ["SLACK_TOKEN"]
+    return os.environ['SLACK_TOKEN']
 
 
-def get_all_users() -> list:
-    return json.loads(os.environ.get("USERS"))
+def get_users(client: WebClient) -> list:
+    users = json.loads(os.environ.get('USERS'))
+    return get_filtered_users(users, client)
+
+
+def get_filtered_users(users: list[str], client: WebClient) -> list[str]:
+    filtered_users = filter(lambda user: is_included_user(user, client), users)
+    return list(filtered_users)
+
+
+def is_included_user(user: str, client: WebClient) -> bool:
+    user_info = client.users_info(user=user)['user']
+    absence_emojis = [':palm_tree:', ':face_with_thermometer:']
+    if user_info['deleted'] is True:
```

Comment:
```suggestion
    if user_info['deleted']:
```
Der Vergleich mit True ist implizit schon da.